Author: <hesso@pool.math.tu-berlin.de>
Description: This is not getopt() and we don't have two leading dashes, so
parse and match the long options in full.
--- a/screen.c
+++ b/screen.c
@@ -667,6 +667,8 @@
 		    }
 		  break;
 		case 'w':
+		  if (strcmp(ap+1, "ipe"))
+		    exit_with_usage(myname, "Unknown option %s", --ap);
 		  lsflag = 1;
 		  wipeflag = 1;
 		  if (ac > 1 && !SockMatch)
